ecowitt2mqtt/hass.py

```diff
@@ -134,6 +140,20 @@ class EntityDescription:
         device_class=DEVICE_CLASS_TEMPERATURE,
         unit_class=UNIT_CLASS_TEMPERATURE,
     ),
+    DATA_POINT_LIGHTNING: EntityDescription(
```

Comment:
So "lighting" is the distance from the last strike and "lightning_num" is the number of strikes in the last day. So you can simply flip `DATA_POINT_LIGHTNING` with `DATA_POINT_LIGHTNING_NUM` and vise versa. This would correct the icon and units.

If you want to keep them alphabetically then we just need to flip the Entity Description code blocks.